src/FSharpLint.Client/LSPFSharpLintServiceTypes.fsi

```diff
@@ -0,0 +1,50 @@
+module FSharpLint.Client.LSPFSharpLintServiceTypes
+
+type FSharpLintResponseCode =
+    | ToolNotFound = 1
+    | FileNotFound = 2
+    | FilePathIsNotAbsolute = 3
+    | CancellationWasRequested = 4
+    | DaemonCreationFailed = 5
+    | Version = 6
```

Comment:
@MrLuje what does Version here mean exactly? maybe you mean VersionMismatch ?

Reply:
No, the opposite. The happy path of the `Version` call
```fsharp
abstract VersionAsync: VersionRequest * ?cancellationToken: CancellationToken -> Task<FSharpLintResponse>
```
Maybe we need a better wording for it ? `DaemonVersion`, `CurrentVersion` or `ToolVersion` ?

Reply:
Well, I thought it was an error because all the other enum members are errors. If it's the happy path, shouldn't it map to 0 instead of 6?

Reply:
At some point, there will be more, like `LintedFile` for LintRequest so I don't think the order really matter :)

Reply:
Oh I see, I thought these were proc exitCodes! Then, do you mind we make the errors negative and non-errors positive? Something like:

```
type FSharpLintResponseCode =
    | ToolNotFound = -5
    | FileNotFound = -4
    | FilePathIsNotAbsolute = -3
    | CancellationWasRequested = -2
    | DaemonCreationFailed = -1
    | Version = 0
```

Then the next non-error response codes can continue with 1, 2...

Reply:
I would even add a prefix to them to make it crystal clear:

```
type FSharpLintResponseCode =
    | ErrToolNotFound = -5
    | ErrFileNotFound = -4
    | FilePathIsNotAbsolute = -3
    | ErrCancellationWasRequested = -2
    | ErrDaemonCreationFailed = -1
    | OkVersion = 0
```

Reply:
oh, and renaming Version was also a good idea from you, so `OkCurrentDaemonVersion`? :)

Reply:
Looks good, go for negative, Ok/Err & `OkCurrentDaemonVersion` 👍